tests/expose-values.js

```diff
@@ -0,0 +1,43 @@
+//Test things dealing with the initialValues prop
+import React from 'react';
+import { spy } from 'sinon';
+import Form from '../src/form';
+import { shallow } from 'enzyme';
+
+const Input = ({ error, ...props }) =>
+  error ? <p className="error">{error}</p> : <input {...props} />;
+
+test('Form passes values to onValues', () => {
+  const onValues = spy();
```

Comment:
jest has [its own spy](https://facebook.github.io/jest/docs/jest-object.html#jestspyonobject-methodname), do you just prefer `sinon`'s implementation?